### ISSUE TITLE
onkeydown and onkeypress do not trigger actions

### ISSUE THREAD

**Issue by ashrafhasson** [kaliber5/ember-bootstrap-power-select#16]
Hi there,

I'm trying to have power-select's 'onkeydown' or 'onkeypress' event fire an action on my controller but it doesn't seem to be triggering anything when used in ember-bootstrap's form.element's yielded control block:

```htmlbars
// templates/some-end-point.hbs
{{#form.element controlType="power-select" options=optionsList optionLabelPath="name" property="someModelWeBelongTo" as |el|}}
  {{#el.control
    searchField="id"
    onkeypress=(action 'updateOptionsList')
    renderInPlace=true as |op|}}
      {{op.name}} ---  <span class="op-class"> {{op.id}} </span>
  {{/el.control}}
{{/form.element}}
```

`updateOptionsList` is not fired when pressing any key inside the search input box of the select element. It doesn't appear to be receiving the event at all.

Here's my action method:

```javascript
// controllers/some-end-point.js
actions: {
  updateAccountsPayableList(dropdown, e) {
    if (e.keyCode !== 13) { return; }
    let text = e.target.value;
    console.log('here <<<');
    console.log(text);
    console.log('here >>>');
  }
}
```

I'm not sure where I'm going wrong here but maybe someone can point me in the right direction?

Thanks for this plugin, btw!

**Comment by ashrafhasson** [comment]
I should add that I've tested wiring the `onkeydown` and `onkeypress` events to controller actions directly in `ember-power-select` it works fine.

**Comment by simonihmig** [comment]
Should be fixed by #31
